modules/generic_worker/templates/generic-worker.plist.erb

```diff
@@ -12,7 +12,7 @@
         <string>/usr/local/bin/run-generic-worker.sh</string>
         <string>run</string>
         <string>--config</string>
-        <string>/etc/generic-worker.config</string>
+        <string>/etc/generic-worker/config</string>
```

Comment:
Does this need to go back to a dot to match the reference in  modules/generic_worker/manifests/multiuser.pp (https://github.com/mozilla-platform-ops/ronin_puppet/pull/75/files#diff-fb9e83eb4fe169b0d24255d1dd4a5180R82)

Reply:
nm, you pushed a new commit switching to using this /etc/generic-worker directory. 👍

Reply:
thx. I see the working dir paths are all matching now for multiuser. It is a little confusing to have the config in different places with similar names.